_spinnaker/policy_engine.md

```diff
@@ -19,13 +19,16 @@ Armory recommends the following versions for the Policy Engine:
 * Spinnaker 2.16.0 or later
```

Comment:
This version needs to be bumped later - 2.18.1 most likely?

Reply:
I think we should bump it to the lowest version that supports both pipeline save and run time. 

That 2.16.0 number was pulled out of thin air after talking to Beth.

Reply:
for pipeline that's 2.16
for runtime that's in 2.18.1-rcXXX (currently), i'm using 2.18.1-rc711

Reply:
How often do upgrade to RC builds? My gut is to point them to 2.19.0 instead